python27-sys/Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://github.com/dgrunwald/rust-python27-sys"
 repository = "https://github.com/dgrunwald/rust-python27-sys.git"
 license = "Python-2.0"
 authors = ["Daniel Grunwald <daniel@danielgrunwald.de>"]
+links = "python27"
```

Comment:
Why is this necessary? Shouldn't the build script output the linker flags? Especially since there's `python27` and `python2.7` depending on platform...